docs/skopeo-sync.1.md

```diff
@@ -68,6 +68,8 @@ Print usage statement.
 
 **--append-suffix** _tag-suffix_ String to append to destination tags.
 
+**--top-refs** Only synchronize for top latest refs after sorting by [Semver](https://semver.org/) provided by [go-version](https://github.com/hashicorp/go-version) lib. Fail if image's refs not follow Semantic versioning.
```

Comment:
Users don‘t care about the implementation library.

---

If this actually only sorts by tags, it should say tags and not refs.

---

Further updates might be necessary depending on how exactly the scope of the filtering ends up being implemented.